python-wrapper/src/neo4j_viz/nvl.py

```diff
@@ -51,14 +51,17 @@ def render(
         js_code = f"""
         var myNvl = new NVLBase.NVL(
             document.getElementById('{container_id}'),
+            document.getElementById('tooltip'),
```

Comment:
I noticed an issue that when having several NVL instances, the tooltip always shows in the first one:

![Screenshot from 2025-04-22 16-47-14](https://github.com/user-attachments/assets/2ec4457f-9eb7-4c35-a5a7-81add50ec8bf)

This is because we're re-using the some `"tooltip"` DOM id, which should be unique. We can resolve this by adding the `container_id` to the string:

```suggestion
            document.getElementById('{container_id}-tooltip'),
```

Reply:
Nice catch!

Reply:
done